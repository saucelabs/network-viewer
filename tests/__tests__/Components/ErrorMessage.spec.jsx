import React from 'react';
import { shallow } from 'enzyme';

import ErrorMessage from './../../../src/Components/ErrorMessage';

describe('ErrorMessage', () => {
  const props = {
    title: 'Error message title',
  };

  it('renders without crashing', () => {
    const element = shallow(<ErrorMessage {...props} />);
    expect(element).toMatchSnapshot();
  });
});
