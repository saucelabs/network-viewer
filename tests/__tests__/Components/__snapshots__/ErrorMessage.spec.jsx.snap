// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders without crashing 1`] = `
<div>
  <h4>
    Error message title
  </h4>
</div>
`;
